common/v2/services/EthService/network/providerHandler.ts

```diff
@@ -38,6 +52,7 @@ export class ProviderHandler {
         data: this.requests.getTokenBalance(address, token).params[0].data
       })
       .then(data => ERC20.balanceOf.decodeOutput(data))
+      .then(({ balance }) => balance)
```

Comment:
Am I missing something or this line leads to balance being undefined in next `.then()` call when `balance` is destructured again.

Since getTokenBalance is using the same calls as getRawTokenBalance it might be good to reuse it.
```
/* Tested */
  public getTokenBalance(address: string, token: Asset): Promise<string> {
    return this.getRawTokenBalance(address, token).then(balance => {
      if (token.decimal) {
        return baseToConvertedUnit(balance, token.decimal);
      }
      return baseToConvertedUnit(balance, 18);
    });
  }
```

Reply:
Done